README.rst

```diff
@@ -43,6 +53,8 @@ Obviously, TweetDeck authors can change anything anytime, which may or may not b
 .. image:: https://travis-ci.org/honzajavorek/schedule-tweet.svg?branch=master
     :target: https://travis-ci.org/honzajavorek/schedule-tweet
 
+The same tests run every time a Pull Request is created. Individual commit pushes are not tested to prevent collisions.
```

Comment:
@honzajavorek IMHO pull request don't have access to encrypted Travis vars, or do they?

Reply:
They have. But only mine. I didn't realize that. Basically, the current tests are only good as "nightly". It probably doesn't make sense to run them at any other times.